docs/units/equivalencies.rst

```diff
@@ -201,28 +234,35 @@ its arguments the |quantity| for the spectral location. For example::
     >>> b.to(a.unit, u.spectral_density(w))  # doctest: +FLOAT_CMP
     <Quantity 1. erg / (cm2 s)>
 
-Brightness Temperature / Surface Brightness Equivalency
--------------------------------------------------------
+.. EXAMPLE END
+
+Brightness Temperature and Surface Brightness Equivalency
+---------------------------------------------------------
 
 There is an equivalency between surface brightness (flux density per area) and
-brightness temperature.  This equivalency is often referred to as "Antenna
-Gain" since, at a given frequency, telescope brightness sensitivity is
-unrelated to aperture size, but flux density sensitivity is, so this
-equivalency is only dependent on the aperture size.  See `Tools of Radio
-Astronomy
-<https://books.google.com/books?id=9KHw6R8rQEMC&pg=PA179&source=gbs_toc_r&cad=4#v=onepage&q&f=false>`__
+brightness temperature. This equivalency is often referred to as "Antenna Gain"
+since, at a given frequency, telescope brightness sensitivity is unrelated to
+aperture size, but flux density sensitivity is, so this equivalency is only
+dependent on the aperture size. See `Tools of Radio Astronomy
+<https://books.google.com/books?id=9KHw6R8rQEMC&pg=PA179&source=gbs_toc_r&cad=4#v=onepage&q&f=false>`_
 for details.
 
 .. note:: The brightness temperature mentioned here is the Rayleigh-Jeans
           equivalent temperature, which results in a linear relation between
           flux and temperature. This is the convention that is most often used
           in relation to observations, but if you are interested in computing
-          the *exact* temperature of a planck function that would produce a
+          the *exact* temperature of a Planck function that would produce a
           given flux, you should not use this equivalency.
 
+Examples
+^^^^^^^^
+
+.. EXAMPLE START: Converting Brightness Temperature and Surface Brightness
+   Equivalency
+
 The `~astropy.units.equivalencies.brightness_temperature` equivalency requires
-the beam area and frequency as arguments.  Recalling that the area of a 2D
-gaussian is :math:`2 \pi \sigma^2` (see `wikipedia
+the beam area and frequency as arguments. Recalling that the area of a 2D
+Gaussian is :math:`2 \pi \sigma^2` (see `wikipedia
```

Comment:
You may be amused by our footnote 2 in https://arxiv.org/abs/2005.14199 😄

Reply:
"2   In this Note, we obey physics and astronomy conventions and refer to the normal pdf as the Gaussian pdf. We realize that it is irresponsible to name things after people when those same things also have generic, descriptive names. On the other hand, “normal” isn’t the finest name either. Perhaps Gaussian pdfs should be called “central” since they are produced by the central limit theorem. Anyway, we will continue with the name “Gaussian” despite our own reservations. We apologize to our reader." 😄 👍

Reply:
Nice! (and also a nice note, very instructive!)